### PR TITLE
Remove unnecessary ampersands at end of url

### PR DESCRIPTION
**Pull request by OleeL**
Hey, I just came across your crate today and ran into an issue with it regarding trailing ampersands after params

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -100,6 +100,7 @@ impl URLBuilder {
             for (param, value) in self.params.iter() {
                 url_params.push_str(format!("{}={}&", param, value).as_str());
             }
+            url_params.pop();
```
Comment (Evalir) [inline src/lib.rs:105]: Let's leave a comment here explaining why we need to always `pop` at the end—it's not obvious at first glance

**Review by Evalir — APPROVED** [review]
Thanks for the PR! Really appreciate it 🙏 

Just a comment:

File: src/lib.rs (hunk 1 above)
Comment (OleeL) [reply src/lib.rs:105]: That's understandable. I've just added a comment.

**Comment by Evalir** [comment]
Thanks! Will fix the test and publish ASAP.

**Pull request merged**